app/src/main/res/values/strings.xml

```diff
@@ -188,7 +188,7 @@
     <string name="artists">Artists</string>
     <string name="yes">Yes</string>
     <string name="later">Later</string>
-    <string name="disabled">Disabled</string>
+    <string name="disabled">Off</string>
```

Comment:
From a developer perspective I would expect the string behind "disabled" to be disabled and not off.

Reply:
Is it something that can be on _and_ disabled?